widget/list.go

```diff
@@ -109,10 +109,21 @@ func (l *List) SetItemHeight(id ListItemID, height float32) {
 }
 
 func (l *List) scrollTo(id ListItemID) {
-	if l.scroller == nil {
+	if l.scroller == nil{
 		return
 	}
-	y := (float32(id) * l.itemMin.Height) + (float32(id) * theme.Padding())
+
+	separatorThickness := theme.Padding()
+	y := float32(0)
+	for i := 0; i < id; i++ {
```

Comment:
This will certainly impact the speed of scrollTo on large list as it as discussed in other case. Can you put the bypass logic in place here too for now?